base/src/commonMain/kotlin/utils/Locale.kt

```diff
@@ -80,6 +80,11 @@ expect class Locale {
      */
     val variantCode: String
 
+    /**
+     * Returns true if the [Locale] uses the metric system.
+     */
+    val usesMetricSystem: Boolean
```

Comment:
I dont think this should be a boolean. It should be an enum that indicates the measurement system used.
This does not account properly for cases like the UK where there is partial metric.